app/services/databases/firestore/leetcode_submissions.py

```diff
@@ -76,6 +77,18 @@ def get_user_submission_for_problem(self, uuid: str, problem_id: str):
             print(f"Error in get_user_submission_for_problem for user {uuid}: {e}")
             raise e
 
+    def get_all_user_uuids(self) -> list:
+        '''
+        Get all user uuids
+        '''
+        collection_ref = self.get_collection(self.uuid_collection)
+        try:
+            docs = collection_ref.stream()
+
+            return [doc.id for doc in docs]
+        except Exception as e:
+            print(f"Error in get_all_user_uuids: {e}")
+            raise e
```

Comment:
@tin159iop Not sure why this doesn't get all the uuids from the collection? Not sure if you know why?